packages/js/src/workouts/tailwind-components/Stepper.js

```diff
@@ -138,7 +141,7 @@ function TailwindStep( { step, stepIndex, isLastStep, saveStep, activeStepIndex,
 				</span>
 				{ /* Name and description. */ }
 				<span className="yst-ml-4 yst-min-w-0 yst-flex yst-flex-col yst-self-center">
-					<span className={ "yst-text-xs yst-font-semibold yst-tracking-wide yst-uppercase " + nameClassNames }>
+					<span className={ "yst-text-xs yst-font-[650] yst-tracking-wide yst-uppercase " + nameClassNames }>
```

Comment:
What does this arbitrary value here resolve to? I can't track it. 
Also, if this is at the "theme" level (in other words, will there be a lot of headings with this font-size?), we should probably extend the font class, not use arbitrary values.